packages/generators/utils/language.ts

```diff
@@ -0,0 +1,128 @@
+export enum LangType {
+	ES6 = "ES6",
+	Typescript = "Typescript",
+}
+
+interface ModuleRule extends Object {
+	include?: string[];
+	exclude?: string[];
+	loader: string;
+	options?: {
+		plugins: string[];
+		presets: Preset[][];
+	};
+	test: string;
+}
+
+type Preset = string | object;
```

Comment:
I would use the types coming from `webpack`: https://github.com/webpack/webpack/blob/master/declarations/WebpackOptions.d.ts#L82